intranet/static/js/common.js

```diff
@@ -31,25 +30,159 @@ $(function() {
-runEgg = function(q) { if(q == "do a barrel roll") { setTimeout(function() { ytwin("mv5qzMtLE60"); }, 1); setTimeout(function() { $("body").append("<style>@-webkit-keyframes roll { from { -webkit-transform: rotate(0deg) } to { -webkit-transform: rotate(360deg) } } @-moz-keyframes roll { from { -moz-transform: rotate(0deg) } to { -moz-transform: rotate(360deg) } } @keyframes roll { from { transform: rotate(0deg) } to { transform: rotate(360deg) } } body {-moz-animation-duration: 4s;-moz-animation-iteration-count: 4;-moz-animation-name: roll;-webkit-animation-name: roll; -webkit-animation-duration: 4s; -webkit-animation-iteration-count: 4;animation-name: animation-duration: 4s; animation-iteration-count: 4;}</style>"); }, 2000); } else if(q == "asteroids") { var KICKASSVERSION='2.0';var s = document.createElement('script');s.type='text/javascript';document.body.appendChild(s);s.src='//hi.kickassapp.com/kickass.js'; $("body").append("<style>div#kickass-menu.KICKASSELEMENT {display: none !important;top: -9999px;left: -9999px;} </style>"); } else if(q == "d is for dogecoin") { s=document.createElement("script"),s.src="//wogloms.com/dogeify.js?ion";document.body.appendChild(s); } else if(q == "turn down for what") { $(".title h1").dblclick();setTimeout(function(){var n = $(".title h1 .letter-n");if(n.length>0)n.dblclick(); else eggTdfw()},100); } else if(q == "libdoge") { var s = document.createElement('script');s.type='text/javascript';s.src='//raw.github.com/jwoglom/libdoge/master/libdoge/bookmarklet-min.js';document.body.appendChild(s); } else return false; return true; }
+function ytwin(id) {
+    $("body").click(function() {
+        $("iframe#ytwin").remove();
+    }).append('<iframe id="ytwin" style="position:fixed;top:50%;left:50%;width:640px;height:480px;margin:-240px -320px" width="640" height="480" src="https://www.youtube.com/embed/' + id + '?autoplay=1&loop=0" frameborder="0" allowfullscreen></iframe>');
+};
+try {
+    new(function(callback) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Do not use 'new' for side effects.](https://www.codacy.com/app/tjcsl/ion/file/2743307438/issues/source?bid=3176160&fileBranchId=3355598#l55)